### ISSUE TITLE
Microchip Code Configurator / Harmony3 Support

### ISSUE THREAD

**Issue by timscheuermann** [callwyat/mplab-extension#28]
There are standalone options of Harmony and MCC available.
What about adding those, so the extension can create MCC- / Harmony-generated code by itself and we won't need to run MPLABX for this?

I tried the standalone-versions by myself and it looked pretty promising, unfortunately I have no knowledge on how to implement external tools in an vscode extension.

Look at these links for reference:
-> **Harmony Standalone (preinstalled with Harmony):** https://microchip-mplab-harmony.github.io/mhc/doc/readme_mhc_important_files.html
-> **MCC Standalone:** https://gallery.microchip.com/packages/MCC-Standalone-(Windows)/

**Comment by callwyat** [comment]
This sounds like an interesting idea! However, I think it is outside of the scope of this extension. I would recommend creating a new extension that was focused on integrating MPLABX Extensions with VSCode. When I started this extension, I too had little to no knowledge to get started, but I was able to find some [documentation](https://code.visualstudio.com/api/get-started/your-first-extension) that gave me the confidence to get going. After that, it just been a lot of googling for things like "VSCode Api command", "NodeJS how to read a file", "Typescript if statement". I hope that you are able to make something amazing!
